__tests__/api-endpoints.test.js

```diff
@@ -0,0 +1,61 @@
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const request = require("supertest");
+const db = require("../db/connection");
+const fs = require("fs/promises");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET: /api", () => {
+  test("GET 200: returns an JSON object containing required API endpoint data", () => {
+
+
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const endpoints = response.body;
+
+        const expectedOutput = {
+          "GET /api": {
```

Comment:
Rather than having this variable of expectedOutput, you can require in the endpoints.json file and then expect the response to equal it. 👍 This will just save you having to come back and edit this test every time you change the file.